crates/rari-doc/src/templ/templs/csssyntax.rs

```diff
@@ -56,6 +56,10 @@ pub fn csssyntax(name: Option<String>) -> Result<String, DocError> {
 
     let sources_prefix = l10n_json_data("Template", "formal_syntax_footer", env.locale)?;
 
+    if env.browser_compat.len() > 1 {
+        warn!("Multiple browser-compat entries found. CSS formal syntax will be rendered using the first entry as the scope: {}", env.browser_compat.first().unwrap());
```

Comment:
Nit: I think we can log all entries, to make it easier for the reader to understand what the other entries are.

```suggestion
        warn!("Multiple browser-compat entries found. CSS formal syntax will use the scope of the first entry: {}", env.browser_compat);
```